.github/workflows/build-docs.yml

```diff
@@ -86,17 +86,29 @@ jobs:
 
       - name: Generate docs
         continue-on-error: true
+        env:
+          VERSION: ${{ github.event.release.tag_name }}
         run: |
          echo "Custom work on generating docs can go here."
+          
          # Run cn docs 
          sudo bash ./automation/docs/generated-cn-docs.sh
+          
          # Generate property docs and push to main
          sudo bash ./automation/docs/generate-property-docs.sh 
          cd docs
          git add . || echo "Nothing to add"
          git commit -a -S -m "docs: automated property doc generation" || echo "Nothing to commit"
          git push
          cd ..
+          
+         # Replace release number markers with actual release number
+         cd docs
```

Comment:
add a condition here. If the VERSION is empty then you shouldn't run the substitution because this will cause it to be replaced by an empty string which then becomes difficult to fix.

Reply:
Added condition check.